openmetadata-ui/src/main/resources/ui/src/components/common/UserTeamSelectableList/UserTeamSelectableList.component.tsx

```diff
@@ -119,15 +120,21 @@ export const UserTeamSelectableList = ({
 
         setTeamPaging({ total: res.data.hits.total.value });
 
-        return { data, paging: { total: res.data.hits.total.value } };
+        return {
+          data,
+          paging: {
+            total: res.data.hits.total.value,
+            after: toString(afterPage + 1),
+          },
+        };
       } catch (error) {
         return { data: [], paging: { total: 0 } };
       }
     } else {
       try {
         const { data } = await searchData(
           WILD_CARD_CHAR,
-          isNumber(after) ? after : 1,
+          isNaN(afterPage) ? 1 : afterPage,
```

Comment:
We are already checking for isNaN above, so we don't need to check it again here.

```js
const afterPage = isNaN(Number(after)) ? 1 : Number(after);
```